src/Logging.cpp

```diff
@@ -202,3 +202,15 @@ void mc2_log_error(const char *format, ...) {
         va_end(ap);
     }
 }
+
+// mc2: 0x006186B0 Based on modified to remove calls to debug break
```

Comment:
I don't consider removing debug breaks to be "modifying". The debug breaks are essentially just breakpoints (AFAIK), and it's simple enough to use the debugger to do the same thing. I'd/I'll remove this comment.